Source/Android/Harvest/app/src/main/java/za/org/samac/harvest/Stats_Graph.java

```diff
@@ -346,6 +346,9 @@ private LineData getDataFromString(String response){
                 entityNames = functionResult.names(); //to iterate through the top level entities
                 if (entityNames != null) {
                     for (int i = 0; i < entityNames.length(); i++) {
+                        if(entityNames.get(i).equals("exp")){
+                            break;
```

Comment:
Wouldn't break here possibly skip some items? Wouldn't it be better to use continue?

PS: its 'exp' for 'expected'